activerecord/test/schema/schema.rb

```diff
@@ -541,17 +536,10 @@
     t.integer  :salary, default: 70000
     t.references :firm, index: false
     t.integer :mentor_id
-    if supports_datetime_with_precision?
-      t.datetime :legacy_created_at, precision: 6
-      t.datetime :legacy_updated_at, precision: 6
-      t.datetime :legacy_created_on, precision: 6
-      t.datetime :legacy_updated_on, precision: 6
-    else
-      t.datetime :legacy_created_at
-      t.datetime :legacy_updated_at
-      t.datetime :legacy_created_on
-      t.datetime :legacy_updated_on
-    end
+    t.datetime :legacy_created_at
+    t.datetime :legacy_updated_at
+    t.datetime :legacy_created_on
+    t.datetime :legacy_updated_on
```

Comment:
Should this (and other tests) include `precision: 6`, assuming `supports_datetime_with_precision?` is `true` for every adapter?

Reply:
Most `precision: 6` was for MySQL which recognizes implicit precision as 0 precision, unlike other adapters.
Since #42297 set `precision: 6` by default for datetime columns, we can test subseconds even without explicit `precision: 6`.